### PR TITLE
Add Passenger recipe

### PR DESCRIPTION
**Pull request by netwarex**
Closes: #2889

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

File: recipes/passenger/templates/passenger.js
Comment (beerose) [inline recipes/passenger/templates/passenger.js:5]: I think we should use a `blitz` bin instead

**Review by beerose — COMMENTED** [review]
Thanks for your PR!

I'm not sure about the `passenger.js` script though. Left a comment there.

File: recipes/passenger/templates/passenger.js
Comment (netwarex) [reply recipes/passenger/templates/passenger.js:5]: Yes, I tried first that approach, however it does not work because it spawns next.js bin and Passenger stucks with that and created multiple instances without mapping for itself :/

Comment (flybayer) [reply recipes/passenger/templates/passenger.js:5]: Ok we can use the next bin for now. Only limitation is currently next bin does not support custom server but blitz build does. However, soon that will be moved into the next bin and it will fully work

Comment (netwarex) [reply recipes/passenger/templates/passenger.js:5]: Thanks @flybayer :) Another approach I can imagine for the future that the next.js bin is not spawned but required and enhanced with the arguments blitz needs.

**Comment by netwarex** [comment]
Thanks @beerose! The `passenger.js` topic has been resolved :)

**Comment by beerose** [comment]
Hey @netwarex! I'm going to merge this PR, but could you update your branch with the latest canary? Usually, I'd do it myself, but I think the option to update this branch by maintainers is disabled.

**Comment by netwarex** [comment]
Hi @beerose I can't see this option in this merge request, but I have updated with the latest changes the branch :)

File: recipes/passenger/package.json
Comment (beerose) [inline recipes/passenger/package.json:26]:
```suggestion
    "@blitzjs/installer": "0.41.4"
```

**Comment by beerose** [comment]
Hey @netwarex, I'm sorry about that, but the branch is not updated with canary. Could you update it again and I'll try to merge it as soon as I can so that nothing else gets merged in the meantime?
Unfortunately, I can't push to your branch, so can't do it myself.

**Comment by netwarex** [comment]
Hey @beerose! I did it again! :)

File: recipes/passenger/package.json
Comment (beerose) [inline recipes/passenger/package.json:26]:
```suggestion
    "@blitzjs/installer": "0.42.4"
```

**Review by beerose — COMMENTED** [review]
One last thing 🙏

**Comment by netwarex** [comment]
Done ;)

**Review by beerose — APPROVED** [review]

**Pull request merged**
